articles/connections/database/custom-db/templates/change-password.md

```diff
@@ -0,0 +1,534 @@
+---
+description: Custom DB script templates for changing a user's password
+toc: true
+topics:
+    - connections
+    - custom-database
+contentType: how-to
+useCase:
+    - customize-connections
+---
+# Custom Database Script Templates: Change Password
```

Comment:
this doc means that we should update it every time some script gets updated. are we sure we want to do that? did someone ask for it specifically? wouldn't it make more sense to have a revert button in the dashboard?